### ISSUE TITLE
Add print CSS stylesheet

### ISSUE THREAD

**Issue by Bisaloo** [ropensci/roweb2#526]
Currently, if one tries to print blog posts from ropensci.org, it looks really bad.

This could be fixed by adding a CSS stylesheet for printing:

```html
<link rel="stylesheet" href="stylesheet_print.css" media="print">
```

I'm not very good with design decisions but I guess it should decrease margins, possibly remove the `#footer` excepted for the logo and license.

I'm not sure I'll have much time for this in the coming weeks but if no one wants to tackle this, I can try and design a very basic version of this stylesheet.

**Comment by sckott** [comment]
thanks @Bisaloo 

I'm not really a web person, so can't comment on feasibility of that change here, but have you tried pandoc to convert the html files of your posts to pdf?  not sure if it will work, but might be a faster route to get what you need?

**Comment by Bisaloo** [comment]
Yeah, this is what I've done in the mean time. I've grabbed the md file from this repo and just compiled it to pdf with pandoc (it's probably a bit more robust than doing html -> pdf (?)).

I lost the nice rOpenSci header and logo though but that'll do for now.

**Comment by Bisaloo** [comment]
At the moment, I am using the [eisvogel pandoc latex template](https://github.com/Wandmalfarbe/pandoc-latex-template) and I modified the yml header in the following manner:

```
titlepage: true
titlepage-color: "6FAEF5"
titlepage-text-color: "F9F9F9"
titlepage-rule-color: "F9F9F9"
logo: "ropensci.pdf"
listings-no-page-break: true
```

which gives this titlepage which uses rOpenSci color chart and logo:

![Screenshot from 2019-10-28 19-16-58](https://user-images.githubusercontent.com/10783929/67705620-9d020f00-f9b7-11e9-81cf-229ff5e1f183.png)

I'm just not 100% sure it's okay to use your graphical chart for this, even though the content of the document is hosted on your website.

What do you think? If that's something you're not comfortable with, please feel free to let me know, I completely understand :)

**Comment by karthik** [comment]
@Bisaloo This seems fine for the purpose you describe. We are aware of the print style issues and are working on a major website redesign that will fix these and other issues.
